### DISCUSSION TITLE
Is there a validator?

### DISCUSSION THREAD

**Discussion by buhtz** [ximion/appstream#584] [Q&A] [answered]
I will start to create AppStream data for [Back In Time](https://github.com/bit-team/backintime) because I am upstream maintainer.
It could help me if there would be an validator service or application to check if my xml file is correct or not. Kind of a linter or w3c validator.

**Comment by buhtz** [comment, marked as answer]
After the online generator finished there are some more infos comming up.

    appstreamcli validate *.xml

Be aware that the online generator to produce invalid content. AppStreamCli is not satisfied.

**Reply by ximion** [reply]
Which online generator did you use? There shouldn't be any bad data generated...

In any case, you found the right tool for validation :-)
